Assets/HoloToolkit/Input/Scripts/FocusedObjectMessageSender.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using HoloToolkit.Unity;
+
+public class FocusedObjectMessageSender : MonoBehaviour
```

Comment:
@NeerajW @jwittner @keveleigh So what's the consensus? Do we want add a separate class or add a convenience method to GazeManager?